### ISSUE TITLE
CONFIGURE should check if already locally installed in ./R

### ISSUE THREAD

**Issue by gjwgit** [mlhubber/mlhub#72]
For the audit model, every time I run configure it re-installs all models into ./R even if already there.

**Comment by gjwgit** [comment]
In fact, maybe version checking is not functional:
```
$ grep -i Version: ~/R/x86_64-pc-linux-gnu-library/3.5/rpart.plot/DESCRIPTION 
Version: 3.0.6
$ grep -i Version: ~/.mlhub/audit/R/rpart.plot/DESCRIPTION 
Version: 3.0.6
```

**Comment by simonzhaoms** [comment]
Fixed in commit 6dc18ac

**Comment by simonzhaoms** [comment]
However, we still needs to figure out how to avoid download files already exist.
Because only after we download the file can we check if they are the same.
